.travis/before_script.sh

```diff
@@ -1,14 +1,4 @@
 #!/bin/bash
 set -ev
 
-if [ "$TRAVIS_BRANCH" == "master" ]; then
-    OPTIONS="--type daily --branch mozilla-central"
-elif [ "$TRAVIS_BRANCH" == "mozilla-aurora" ]; then
-    OPTIONS="--type daily --branch mozilla-aurora"
-elif [ "$TRAVIS_BRANCH" == "mozilla-beta" ]; then
-    OPTIONS="--type release --version latest-beta"
-elif [ "$TRAVIS_BRANCH" == "mozilla-release" ]; then
-    OPTIONS="--type release --version latest"
-fi
-
-mozdownload $OPTIONS
+mozdownload --type tinderbox --branch mozilla-beta
```

Comment:
I have changed this from release to tinderbox to be closer on what we are testing for branch merges. It could still fail if we branch our tests before an appropriate tinderbox build is present, but in general this is safer.